backend/src/main/scala/cromwell/backend/package.scala

```diff
@@ -59,4 +59,16 @@ package object backend {
   }
 
   case class PreemptedException(msg: String) extends Exception(msg)
+
```

Comment:
All the implementations of BID are implementation specific and will live elsewhere, so this can't be sealed and isn't really an ADT anyway.